tests/testthat/test_synthdid.R

```diff
@@ -110,3 +110,58 @@ test_that("invariances hold with default options", {
     }
   }
 })
+
+test_that("treated/control scaling invariances hold with default options", {
+  # Test that four types of invariances hold, for details see
+  # https://github.com/synth-inference/synthdid/issues/43
+  estimators = list(sc_estimate, did_estimate, synthdid_estimate)
+  seed = sample(1:1e6, 1)
+  setup = random.low.rank()
+  T0 = setup$T0
+  N0 = setup$N0
+  T = ncol(setup$Y)
+  exposed = 1:nrow(setup$Y) > N0
+  Y.orig = setup$Y
+
+  for (c in c(1e-6, 0.25, 1e6)) {
+    # 1.
+    # Re-mapping Yit <- Yit + c for exposed t > T0 increases tau by c
+    Y1 <- setup$Y
+    Y1[exposed, (T0+1):T] <- c + Y1[exposed, (T0+1):T]
+    for (estimator in estimators) {
+      estimate = estimator(Y.orig, N0, T0)
+      estimate.shift = estimator(Y1, N0, T0)
+      expect_equal(c(estimate.shift), c(estimate) + c, tol = 1e-2)
```

Comment:
Ideally we'd hope this goes through with a smaller tolerance. It'd be good to understand why it doesn't.

Reply:
Pinging @davidahirshberg here...

Reply:
@davidahirshberg is the Franke-Wolfe solver supposed to be translation invariant? Would fitting on standardized data be a sensible approach?

Reply:
@davidahirshberg is the Franke-Wolfe solver supposed to be translation invariant? Would fitting on standardized data be a sensible approach?

Reply:
Unless there is a bug, this shift should not change the unit or time weights estimated. Neither uses the treated observations in their training.

Reply:
The only change from one call to the next should be in this[ line](https://github.com/synth-inference/synthdid/blob/7ff2bd129383833cf8efe89a0cdcc3a6034dfe7c/R/synthdid.R#L68) of code. 
And some entries of Yc that are never looked at.

Reply:
There was a bug. This translation was affecting the default value of the tuning parameter zeta.omega in sc_estimate. We'd fixed the same bug earlier for synthdid_estimate.